manifests/micro-utilities.json

```diff
@@ -1,5 +1,11 @@
 {
   "moduleReplacements": [
+    {
+      "type": "simple",
+      "moduleName": "call-bind",
+      "replacement": "Use Function.call.bind(v)",
+      "category": "micro-utilities"
+    },
```

Comment:
this is only true for the default export; `call-bind/callBound` is a bit more complex.